android/app/src/main/AndroidManifest.xml

```diff
@@ -36,10 +36,10 @@
                  screen fades out. A splash screen is useful to avoid any visual
                  gap between the end of Android's launch screen and the painting of
                  Flutter's first frame. -->
-            <meta-data
+            <!--<meta-data
```

Comment:
Is this causing some problem? Won't remove Android splash screen?

Reply:
This doesn't cause any problem, but Flutter says this function is deprecated.
https://flutter.dev/docs/development/ui/advanced/splash-screen#migrating-from-manifest--activity-defined-custom-splash-screens

Reply:
Thanks. I will take look, I didn't attempt to run on Android after Flutter update (2.5?)